### PR TITLE
Upwork samarth

### PR DESCRIPTION
**Pull request by samarthdd**
Initial project setup with Flask
Configured Docker and kubernetes , minikube setup.
Added scrapper function to fetch all zip files from the given link  ,download it in local folder and save names in a txt file.

### CODE REVIEW

File: upwork-devs/samarth_dd/virus_share/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Flask ==1.1.2
+gunicorn
```
Comment (belwase) [inline upwork-devs/samarth_dd/virus_share/requirements.txt:2]: please include version

File: upwork-devs/samarth_dd/virus_share/src/file_uploadrer.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Import MinIO library.
+from minio import Minio
+from minio.error import (ResponseError, BucketAlreadyOwnedByYou,
+                         BucketAlreadyExists)
+
+# Initialize minioClient with an endpoint and access/secret keys.
+minioClient = Minio('play.min.io',
+                    access_key='Q3AM3UQ867SPQQA43P2F',
```
Comment (belwase) [inline upwork-devs/samarth_dd/virus_share/src/file_uploadrer.py:8]: Don't include access and secret keys on code. Include environment variables

File: upwork-devs/samarth_dd/virus_share/src/scrapers/virusshare.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+
+
+
+class Scraper(BaseScraper):
+    def __init__(self):
+
+        self.base_url = "https://cloud.corvusforensics.com"
+        self.url = 'https://cloud.corvusforensics.com/s/HDcCHLnQTGBnYBN'
+
```
Comment (belwase) [inline upwork-devs/samarth_dd/virus_share/src/scrapers/virusshare.py:19]: Let's not use more spaces than 2 lines

File: upwork-devs/samarth_dd/virus_share/src/scrapers/virusshare.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        download_path = cwd + "/test"
+        isFile = os.path.isdir(download_path)
+        if (not isFile):
+            os.mkdir(download_path, 0o777);
+        newfile = open('zipfiles.txt', 'w')
+        print("Running script.. ")
+        extension = ".zip"
+        html = None
+
+        browser = webdriver.Chrome()
```
Comment (belwase) [inline upwork-devs/samarth_dd/virus_share/src/scrapers/virusshare.py:33]: python requests works fine for this scraper. so we don't need selenium or browser automation

**Review by belwase — CHANGES_REQUESTED** [review]
Please check my comments.
And also don't include .pyc file on commit.

File: upwork-devs/samarth_dd/virus_share/src/scrapers/virusshare.py (hunk 4 above)
Comment (samarthdd) [reply upwork-devs/samarth_dd/virus_share/src/scrapers/virusshare.py:33]: The data in the mentioned link loads using javascript.So i requests wont fetch tbody data..Will check for beeter solution.

Comment (DinisCruz) [reply upwork-devs/samarth_dd/virus_share/src/scrapers/virusshare.py:26]: Browser automation should be done using pyppeteer

File: upwork-devs/samarth_dd/virus_share/src/scrapers/virusshare.py (hunk 3 above)
Comment (DinisCruz) [reply upwork-devs/samarth_dd/virus_share/src/scrapers/virusshare.py:14]: Please check the GW coding guidelines

**Review by DinisCruz — APPROVED** [review]
I’m going to merge, but please continue working in collaboration with @belwase since his code base is already more evolved

**Pull request merged**